lib/modules/platform/bitbucket/readme.md

```diff
@@ -22,7 +22,7 @@ Let Renovate use your app password by doing _one_ of the following:
 
 Remember to:
 
-- Set the `username` for the bot account
+- Set the `username` for the bot account (which is _not_ your email address). You can find your username through "Personal Bitbucket settings" on the "Account settings" page for your account.
```

Comment:
```suggestion
- Set the `username` for the bot account (which is _not_ your email address). You can find your username by going to the "Personal Bitbucket settings" on your "Account settings" page
```

This may sound a bit silly, but we do not end _list items_ with a full stop. But _normal_ sentences should of course end with a full stop. :wink: 

I'm also rewriting the bit you added to be shorter.

Read the [Renovate style guide](https://github.com/renovatebot/renovate/blob/main/docs/development/style-guide.md) to learn more about the conventions for user-facing text.

Reply:
THanks for the pointer to the Renovate style guide, was not aware of it.

Reply:
```suggestion
- Set the `username` for the bot account (which is _not_ your email address). You can find your username by going to your Bitbucket "Account settings" page and then to "Personal Bitbucket settings"
```

How about this? It's a bit longer, but it lists the steps in more detail now.